### PR TITLE
Added date component helper to readme. Some code improvements.

### PR DESCRIPTION
**Pull request by elland**
Forgot to document it. 👀

### CODE REVIEW

**Comment by elland** [comment]
@3lvis mind adding us as admin to this project? 🤔

**Comment by 3lvis** [comment]
Made everyone Owner 🔥⚡️

File: README.md [hunk 1]
```diff
@@ -81,6 +82,15 @@ With this:
 "hello world".length
 ```
 
+
+## Date
+Easily break down a date into components with our helper method:
+
+```swift
+let date = Date()
+date.components([.hour, .day. year. calendar])
```
Comment (designatednerd) [inline README.md]: Calendar should probably be outside the array (or omitted in this bit of documentation)

Comment (elland) [reply README.md]: Missing some commas, misplaced some periods. Removing the component anyway for clarity. 👍

**Review by designatednerd — APPROVED** [review]

**Comment by elland** [comment]
Tagging these changes as a dot-dot release. 👍

**Pull request merged**